### PR TITLE
Add SameSite cookie attribute support

### PR DESCRIPTION
**Pull request by Exelord**
Adds SameSite support: https://github.com/simplabs/ember-cookies/pull/269

### CODE REVIEW

**Comment by marcoow** [comment]
`SameSite` support was released as part of ember-cookies 0.5.1: https://github.com/simplabs/ember-cookies/releases/tag/0.5.1. Can you update the `ember-cookies` dependency as part of this PR and add some tests?

**Comment by Exelord** [comment]
@marcoow Done! :)

File: package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "broccoli-merge-trees": "^2.0.0 || ^3.0.0",
     "ember-cli-babel": "^6.8.2",
     "ember-cli-is-package-missing": "^1.0.0",
-    "ember-cookies": "^0.3.0 || ^0.4.0",
+    "ember-cookies": "^0.3.0 || ^0.5.0",
```
Comment (marcoow) [inline package.json]:
```suggestion
    "ember-cookies": "^0.5.0",
```

The `sameSite` option is only available with >= 0.5.0

**Comment by jfdnc** [comment]
@marcoow Do we know when this change will make it into a release?

**Comment by jfdnc** [comment]
Looks like it made it into 2.1.1!
kthnx

**Pull request merged**